### PR TITLE
feat(TMRX-499): contractual title author profile

### PR DESCRIPTION
**Pull request by bibicollins1**
### Description 

Create the JobTitle component which renders a tooltip or popover component depending on device size, and shows the contractual title of the author on the author profile page. 

https://nidigitalsolutions.jira.com/browse/TMRX-499

Screenshot 2023-05-15 at 10 37 30

### CODE REVIEW

File: packages/ts-newskit/src/components/misc/job-title/job-title.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      <NewsKitTooltipIcon />
+    </IconButton>
+  </Tooltip>
+);
+
+export const JobTitlePopover: React.FC<{ contractualTitle: string }> = ({
+  contractualTitle
+}) => (
+  <Popover
+    content={<>{null}</>}
```
Comment (ibrahim-kurhan) [inline packages/ts-newskit/src/components/misc/job-title/job-title.tsx:41]: you probably know what you are doing but do you have to give null to content?

File: packages/author-profile/src/author-profile-head-jobtitle.js [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
-const AuthorProfileHeadJobTitle = ({ jobTitle }) => (
-  <AuthorProfileHeadJobTitleContainer
-    role="heading"
-    aria-level="2"
-    style={checkStylesForUnits(styles.jobTitle)}
-  >
-    {jobTitle.toLowerCase()}
-  </AuthorProfileHeadJobTitleContainer>
+const AuthorProfileHeadJobTitle = ({ jobTitle, contractualTitle }) => (
+  <div style={{ display: "flex", alignItems: "center" }}>
```
[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/author-profile/src/author-profile-head-jobtitle.js:9]: Yeah could definitely create a component for it 👍

File: packages/ts-newskit/src/theme/times-web-light/style-presets/popover.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export const popoverStylePresets = {
+  popover: {
+    base: {
+      borderStyle: 'none none none none'
```
[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/ts-newskit/src/theme/times-web-light/style-presets/popover.ts:4]: Good shout will try 👍

File: packages/author-profile/src/author-profile-head-jobtitle.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/author-profile/src/author-profile-head-jobtitle.js:9]: @ibrahim-kurhan So have gone with the same approach as the component currently uses and added jobTitleContainer to the styles file.

**Review by ibrahim-kurhan — APPROVED** [review]

**Review by douglasmik — APPROVED** [review]

File: packages/ts-newskit/src/theme/times-web-light/style-presets/popover.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export const popoverStylePresets = {
+  popover: {
+    base: {
+      borderStyle: 'none'
+    }
+  }
+};
```
[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/ts-newskit/src/theme/times-web-light/style-presets/popover.ts:7]: Is that okay? Happy to go for a different approach if that's better. @douglasmik

[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/ts-newskit/src/theme/times-web-light/style-presets/popover.ts:7]: Okay - thanks!

[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/ts-newskit/src/theme/times-web-light/style-presets/popover.ts:7]: Yeah it's used on both the closeButton and the header part of the panel of the popover.. So not sure if theres something else I can apply to both @douglasmik ?

[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/ts-newskit/src/theme/times-web-light/style-presets/popover.ts:7]: In the docs this is how they handle the style presets for the popover with custom styling story so just kind of copied that approach ```() => (
  <GridPage>
    <GridLayoutPopoverItem
      {...defaultContents}
      overrides={{
        pointer: {stylePreset: 'customPopoverPointer'},
        panel: {stylePreset: 'customPopoverPanel'},
        closeButtonContainer: {
          stylePreset: 'customPopoverCloseButtonContainer',
        },
        header: {stylePreset: 'customPopoverHeader'},
      }}
    >
      <Button overrides={btnOverrides}>Show popover</Button>
    </GridLayoutPopoverItem>
  </GridPage>```

**Pull request merged**